gradle/dependencies.gradle

```diff
@@ -24,8 +24,8 @@ ext {
 
 versions += [
   argparse4j: "0.5.0",
```

Comment:
I didn't bump this because we don't have great testing around our command-line tools and I didn't want to manually check that they all still work.